csi/volumeutils.py

```diff
@@ -866,6 +934,7 @@ def mount_glusterfs(volume, mountpoint, is_client=False):
 
     # Ignore if already glusterfs process running for that volume
     if is_gluster_mount_proc_running(volname, mountpoint):
+        reload_glusterfs(volume)
```

Comment:
This call can raise exception, csi server will crash on one notification failure